### ISSUE TITLE
Wasm perf tests RuntimeError: abort(TypeError: cv.Size is not a constructor).

### ISSUE THREAD

**Issue by jlb6740** [opencv/opencv#19754]
I am attempting to run perf tests using node. I think I built with:

`emcmake python3 platforms/js/build_js.py build_wasm --build_wasm --simd --build_wasm_intrin_test --build_perf `

using emscripten 2.0.15. 

Everything builds fine, with the perf tests located here: 

`opencv/build_wasm/bin/perf`

In fact from here I can launch a server and use a chrome (after enabling flags for "experimental-webassembly-features", "webassembly-simd", and "webassembly-threads") and run a test such as perf_threshold.js

However when I go into the same folder and try to run with node ... first doing "npm install" and then running from that directory with: 

`node --experimental-w
asi-unstable-preview1  --experimental-wasm-simd --experimental-wasm-threads --experimental-wasm-type-reflection --experimental-wasm-mv   perf_imgproc/perf_threshold.js`

I have two issues/questions. (1) One there is a lot of text printed which clogs and takes a long time to scroll .. I assume it is printing out ascii values for some file being processed but I am only guessing. I assume it is not expected behavior because of the user experience but I am hoping for confirmation. (2) The second thing is that there is the same runtime error reported no matter the test `RuntimeError: abort(TypeError: cv.Size is not a constructor)`. I thought my version of node (the very latest release) was using too old a V8 since it was older than Chromes, but even after downloading head the error remains. I assume I am just running wrong, but I followed instructions here:  https://docs.opencv.org/master/d4/da1/tutorial_js_setup.html.

**Comment by jlb6740** [comment]
Reading similar issues it seems this is related to some code executing before this cv.Size is known and that the solution is to wait by calling `main()` inside a wrapper like this:

cv['onRuntimeInitialized'] = () => {
  main();
};

When doing this though it seems this is never called.

**Comment by alalek** [comment]
PRs with adding SIMD to opencv.js: #15371 #18068

**Comment by huningxin** [comment]
@lionkunonly , are you able to help look into this issue? Thanks.

**Comment by lionkunonly** [comment]
Ok, I will check it. In the PR [#18068](https://github.com/opencv/opencv/pull/18068) I do not test the execution of perf tests on Node.js carefully. So I will check it soon.

**Comment by lionkunonly** [comment]
@jlb6740 I have found out the root for two issues: 1. The long text is the code of the opencv.js. It is printed out when the program meets an error. 2. The perf tests are designed for the Web in my PR mainly. So I did not notice the problem about the module export on Node.js. So the issue is produced.

@huningxin Hi, Ningxin. Cloud I send a new PR to solve the issue?

**Comment by huningxin** [comment]
> @huningxin Hi, Ningxin. Cloud I send a new PR to solve the issue?

Yes. Please help fix this issue. Thanks in advance.

**Comment by lionkunonly** [comment]
@huningxin  Hi, Ningxin. I have prepared the new PR to fix it. But it seems that I do not have the permission to send the PR to opencv

**Comment by huningxin** [comment]
> @huningxin Hi, Ningxin. I have prepared the new PR to fix it

Great. Thanks.

> But it seems that I do not have the permission to send the PR to opencv

It worked for you before, https://github.com/opencv/opencv/pull/18068. Any issues you met when send the PR? @alalek may help you.
